.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -29,9 +31,24 @@ when applicable, please provide:
 - [ ] Screenshots of any front-end changes are in the PR description
 - [ ] CHANGELOG.rst is updated for high-level changes
 - [ ] You've added yourself to AUTHORS.rst if you're not there
-- [ ] Deleted any part of the PR template that you didn’t edit, except for checkboxes, which you should diligently check as necessary
 
 # Reviewer Checklist
 
 - [ ] Automated test coverage is satisfactory
 - [ ] PR has been fully tested manually
+
+
+<!--
```

Comment:
Might be worth to just put this at the very top since these instructions are intended to be read first. And since they are comments, the Details will still be the first things visible.